int32 count
---
int32 room 
string how 